docs/sdk_developers/how-to-pin-github-actions.md

```
@@ -0,0 +1,78 @@
+In the repository, navigate to **Releases** and open the **Latest** release.
+
+### Step 3: Open the release tag
+
+Click the release version (for example `v2.14.0`) to open the release details page.
+
+### Step 4: Copy the commit SHA
+
+From the release page, open the commit associated with the release and copy the full 40-character commit SHA.
+
+Example SHA: 20cf305ff2072d973412fa9b1e3a4f227bda3c76
+### Step 5: Update the workflow
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)